Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -324,7 +324,7 @@ private void InitializeServiceLocator()
             //If the Mixed Reality Toolkit is not configured, stop.
             if (ActiveProfile == null)
             {
-                Debug.LogError("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
+                // Debug.LogError("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
```

Comment:
Accidental comment

Reply:
Not accidental. Again related to #6735 - my dev console gets filled up with unhelpful and confusing error messages. With switching profiles, it's expected that the ActiveProfile may be briefly null. Hence, showing an error every time the app switches profiles / loads a new scene is not great. How about we show a warning instead?

Reply:
Not accidental. Again related to #6735 - my dev console gets filled up with unhelpful and confusing error messages. With switching profiles, it's expected that the ActiveProfile may be briefly null. Hence, showing an error every time the app switches profiles / loads a new scene is not great. How about we show a warning instead?